scripts/runtestsuite/runtestsuite.py

```diff
@@ -237,16 +239,28 @@ def report_detailed_failure(self, scene, reference_filepath, output_filepath, lo
 
     def __write_header(self, args):
         script_path = os.path.realpath(__file__)
+        git_hash =  subprocess.check_output(['git', 'rev-parse', 'HEAD'])
+        git_title = subprocess.check_output(['git', 'rev-parse', '--abbrev-ref', 'HEAD'])
```

Comment:
What happens if git is not installed or the appleseed version used is a release (we remove .git from the release) ?

Reply:
Ah, good call, I'll add some checks.